### PR TITLE
Added Accessibility Report

### PR DESCRIPTION
**Pull request by LizHareDogs**
Hello,

I have added the Accessibility Report to the top level directory.

Source: RSEAUNZAccessibility.Rmd
Rendered: RSEAUNZAccessibility.html

Looking forward to everyone's thoughts and questions,
Liz

### CODE REVIEW

File: RSEAUNZAccessibility.Rmd [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+To mitigate this, the Global UseR! Working Group created a 
+[Knowledgebase](https://gitlab.com/rconf/userknowledgebase) 
+as a repository for resources and lessons learned.
+
+
+One way to promote inclusion in a conference is to make sure the leadership
+and organizing committee are diverse. 
+Invite people to join rather than 
+relying on self-nomination. 
+The article referenced below provides further advice about leadership, continuity, and diversity.
```
Comment (rowlandm) [inline RSEAUNZAccessibility.Rmd:332]: Is there an article that this references to?

Comment (rowlandm) [reply RSEAUNZAccessibility.Rmd:332]: On line 332 - The article referenced below provides further advice about leadership, continuity, and diversity.

But there doesn't seem to be an article referenced. Sorry, just making that clearer.

**Comment by rowlandm** [comment]
Hi Liz,

This looks great!

I had one small comment that I added to the markdown document - please let me know if you can see it.

File: RSEAUNZAccessibility.Rmd [hunk 2]
```diff
@@ -0,0 +1,334 @@
...
+Since some deaf people use sign language as their first language, it can
+be an ideal accessibility practice. Interpreters should also have an opportunity to become familiar with the field and its vocabulary. One 
+complication with sign language interpretation is that each country
+has its own sign language. Since this conference is focused on Australia, 
+Auslan interpretation should be considered.
+
+Both live captions and sign language interpreters can be integrated easily 
+into Zoom presentations. The Zoom environment should be set up 
+so the user can make the presenter, the slides, and the interpreter visible
+(not small images).
```
Comment (manodeep) [inline RSEAUNZAccessibility.Rmd:198]: Question: Is this still possible with a full-screen presentation (other than enabling gallery mode and then positioning the speaker and the interpreter suitably)?

**Comment by manodeep** [comment]
@LizHareDogs This is brilliant - thank you very much for your time and effort!

**Comment by LizHareDogs** [comment]
Hi Manodeep,

This is a great question, and as a screen reader user I can't really evaluate the ability to get all the images arranged on the screen in a large enough format. 

I understand that Zoom behaves differently in regular meeting mode than in webinar mode, but there was also just a big Zoom software update, at least in the US.

Apparently the presenter and/or sign language interpreter should attend as participants and the person controlling the meeting can allow the person needing access to "pin" the person they always need to be able to see. Here is some information:
https://www.courts.wa.gov/content/publicUpload/Interpreters/ZOOM%20ASL%20TIPS%20zoom-accessibility-tips.pdf

Hope this helps,
Liz


Liz Hare, PhD
Dog Genetics LLC
***@***.***
http://www.doggenetics.com

**Comment by LizHareDogs** [comment]
Hi Rowland,

Thanks for pointing that out!

The information about UseR!2021 is in the paper that is reference 1. The information about UssR!2022 being less inclusive across the board and not accessible is based on personal experience working on inclusion with the R community and now being about to attend this year's conference.

I'll clarify these things in the document all at once when more people have had opportunities to comment.

Thanks again, 
Liz

Liz Hare, PhD
Dog Genetics LLC
***@***.***
http://www.doggenetics.com

**Comment by orchid00** [comment]
@LizHareDogs I wanted to provide details: we will use zoom events with every session being a zoom meeting, we will not be using zoom webinars for this unconference.

**Comment by orchid00** [comment]
@LizHareDogs minor detail, but important here, could you please change your text to be British English. This is because we are in Australia and the Z are S here. 
Thanks!

**Comment by LizHareDogs** [comment]
Hi all,

Thank you for the feedback!

@rowlandm, I added the reference you mentioned.

@manodeep, I'm sorry I haven't been able to figure out more about whether you can successfully "pin" a speaker or interpreter while screen sharing.

@orchid00, I have spell checked against Australian ENglish and hope all the American spellings are gone.

Please let me know if there are problems or questions.

Liz

**Comment by LizHareDogs** [comment]
no additional comment

**Pull request merged**